databricks/koalas/namespace.py

```diff
@@ -124,7 +124,7 @@ def range(start: int,
 
 
 def read_csv(path, header='infer', names=None, usecols=None,
-             mangle_dupe_cols=True, parse_dates=False, comment=None):
+             mangle_dupe_cols=True, parse_dates=False, comment=None, index_col=None):
```

Comment:
We should put the `index_col` argument betwen `names` and `usecols` to follow pandas'?

Reply:
thanks for comment. i agree with you. i've fixed it! 😃